sync-core/src/test/java/com/cloudant/sync/query/AbstractIndexTestBase.java

```diff
@@ -45,21 +45,23 @@ public void setUp() throws SQLException {
         assertThat(ds, is(notNullValue()));
         im = new IndexManager(ds);
         assertThat(im, is(notNullValue()));
-        db = im.getDatabase();
+        db = TestUtils.getDatabaseConnectionToExistingDb(im.getDatabase());
         assertThat(db, is(notNullValue()));
         assertThat(im.getQueue(), is(notNullValue()));
         String[] metadataTableList = new String[] { IndexManager.INDEX_METADATA_TABLE_NAME };
-        SQLDatabaseTestUtils.assertTablesExist(TestUtils.getDatabaseConnectionToExistingDb(db), metadataTableList);
+        SQLDatabaseTestUtils.assertTablesExist(TestUtils.getDatabaseConnectionToExistingDb(db),
```

Comment:
Since `db` has a connection for the current thread, I think we can just replace `Testutils.getDatabaseConnectionToExisitingDb(db)` in this call to just `db`